### PR TITLE
Added modern theme files; fixed font-face weight values

### PR DESCRIPTION
**Pull request by Blackbaud-PaulCrowder**

### CODE REVIEW

**Comment by Blackbaud-PaulCrowder** [comment]
https://github.com/blackbaud/skyux-theme/issues/109

**Comment by Blackbaud-BobbyEarl** [comment]
Preview build at [https://developer.blackbaud.com/skyux-theme?_sp.spa=pr-modern-theme-styles](https://developer.blackbaud.com/skyux-theme?_sp.spa=pr-modern-theme-styles)

File: skyuxconfig.json [hunk 1]
```diff
@@ -25,6 +25,7 @@
   "app": {
     "styles": [
       "@skyux/theme/css/sky.css",
+      "@skyux/theme/css/themes/modern/styles.css",
```
Comment (Blackbaud-SteveBrush) [inline skyuxconfig.json:28]: Currently, Builder injects the `@skyux/theme/css/sky.css` style sheet automatically (even if the SPA lists it in skyuxconfig.json). Do you think we should update Builder to supply the modern stylesheet too?

Comment (Blackbaud-PaulCrowder) [reply skyuxconfig.json:28]: I want SPAs to be able to opt-in to supporting the new theme, so I'd prefer SPAs that don't support it to not even load the CSS. I guess we could do that via a config option in `skyuxconfig.json`.

**Comment by Blackbaud-BobbyEarl** [comment]
Preview build at [https://developer.blackbaud.com/skyux-theme?_sp.spa=pr-modern-theme-styles](https://developer.blackbaud.com/skyux-theme?_sp.spa=pr-modern-theme-styles)

**Pull request merged**